components/feature/media/src/main/java/mozilla/components/feature/media/notification/MediaNotification.kt

```diff
@@ -60,10 +60,10 @@ internal class MediaNotification(
 private fun MediaState.toNotificationData(context: Context): NotificationData {
     return when (this) {
         is MediaState.Playing -> NotificationData(
-            title = session.titleOrUrl,
-            description = session.url,
+            title = session.getTitleOrUrl(context),
+            description = session.nonPrivateUrl,
```

Comment:
Perhaps it would be more readable to have `val`s at the top of the function?

```kt
private fun MediaState.toNotificationData(context: Context): NotificationData {
  val url = if (private) "" else url
  val icon = if (private) null else icon
  return when (this) {
     ...
```

Reply:
🤷‍♂

Reply:
Just tried changing that. The `session` property is not available up there since this is not available on all `MediaState`s.